### PR TITLE
Removes civilian: Return of the Assistant King

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Long ago, Paradise embarked on a project to become a "commerce station"....without going into that history---it didn't work out and was removed---everything related to it is now gone---save one.

"Civilian". 

This is being removed to bring us in-line with the stereotypical SS13 lore, culture, and back to being a fully fledged science station.

Additionally, this further emphasizes the role of assistant that you're still an employee of the station, still have a job (even if that's loose) and still have a paycheck.

As such, all assistant alt-titles are being removed, and only `assistant` will remain.

:cl: Fox McCloud
tweak: Civilian returns to its rightful position of "Assistant"
/:cl:

### CODE REVIEW

File: code/game/machinery/tcomms/nttc.dm [hunk 1]
```diff
@@ -23,10 +23,9 @@
 		"Cyborg" = "airadio",
 		"Personal AI" = "airadio",
 		"Robot" = "airadio",
-		// Civilian + Varients
+		// Assistant + Varients
 		"Assistant" = "radio",
 		"Businessman" = "radio",
```
Comment (Fox-McCloud) [inline code/game/machinery/tcomms/nttc.dm:28]: need to remove these.

File: code/game/objects/structures/misc.dm [hunk 2]
```diff
@@ -11,9 +11,6 @@
 	anchored = 1
 	density = 1
 
-/obj/structure/signpost/attack_hand(mob/user as mob)
-	to_chat(user, "Civilians: NT is recruiting! Please head SOUTH to the NT Recruitment office to join the station's crew!")
```
Comment (Fox-McCloud) [inline code/game/objects/structures/misc.dm:15]: This was the NT Recruiter job's sign that was on station.

File: code/modules/mob/living/carbon/human/species/plasmaman.dm [hunk 3]
```diff
@@ -135,7 +135,7 @@
 		if("Blueshield")
 			O = new /datum/outfit/plasmaman/blueshield
 
-		if("Assistant", "Tourist", "Civilian", "Businessman", "Trader")
+		if("Assistant", "Tourist", "Businessman", "Trader")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm:138]: also remove theses

File: tgui/packages/tgui/interfaces/CardComputer.js [hunk 4]
```diff
@@ -227,8 +227,8 @@ export const CardComputer = (props, context) => {
                       onClick={() => act("assign", { assign_target: v })} />
                   ))}
                 </LabeledList.Item>
-                <LabeledList.Item label="Civilan">
```
Comment (Fox-McCloud) [inline tgui/packages/tgui/interfaces/CardComputer.js:230]: This was broken in the current build---oops.

**Comment by S34NW** [comment]
Gave me a right heart attack you did

**Comment by SabreML** [comment]
If you're removing the other alt titles as well it'd be good to have that in the changelog/PR description.

**Comment by S34NW** [comment]
> 
> 
> If you're removing the other alt titles as well it'd be good to have that in the changelog/PR description. Since currently it's only mentioning 'Civilian'.

They do mention it:

> As such, all assistant alt-titles are being removed, and only assistant will remain.

**Comment by SabreML** [comment]
> They do mention it:

Ah I missed that, sorry. Just the changelog then.

**Comment by Sikroth** [comment]
Making people have an Assistant job won't make them any more likely to be actually productive, and at least IC there's an excuse to be a lazy bum if you're just an unpaid visitor to the station.

> to bring us in-line with the stereotypical SS13 lore, culture

Isn't most of that "lore and culture" mostly born from low/no rp servers? I'm not saying it's bad but I don't think removing off-duty jobs does much to add to any form of lore and it definitely doesn't add to roleplaying options either.

Like it or hate it a lot of people play the game just to hang out and relax and it makes sense to have a place for that kind of people, I think.

Just my 2 cents anyway, I personally use assistant regardless, I just don't think having the others is bad in any way.

**Comment by LiberatedWaveMan** [comment]
I won't repeat what they said, but Sikroth made some very good points; mainly, changing job titles isn't going to change the gameplay or people's behavior at all.

**Review by dearmochi — APPROVED** [review]

**Pull request merged**